amr-wind/physics/TerrainDrag.H

```diff
@@ -28,14 +28,12 @@ public:
 
     void post_init_actions() override {}
 
-    void post_regrid_actions() override {}
+    void post_regrid_actions() override;
```

Comment:
Any reason why this is changed back to `;` vs `{}`

Reply:
because I actually define this function now with something non-trivial in the cpp file.